mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -2933,7 +2933,7 @@ void hasRightShouldThrowOnUnknownMailbox() {
                     MailboxPath.forUser(USER_1, "notFound"),
                     MailboxACL.Right.Administer,
                     session))
-                .isInstanceOf(MailboxNotFoundException.class);
+                .hasRootCauseExactlyInstanceOf(MailboxNotFoundException.class);
```

Comment:
:-1: we should manage to unwrap the exception in order not to break things.

CF MailboxReactorUtils for blocking.